### DISCUSSION TITLE
When quarkus-mongodb-client will supports MongoDB driver version 5?

### DISCUSSION THREAD

**Discussion by mancioshell** [quarkusio/quarkus#42305] [Community]
[Mongo Driver v5](https://www.mongodb.com/community/forums/t/mongodb-java-driver-5-0-is-released/268913) has been release few months ago. When quarkus will update the driver of the mongo client? Is it in roadmap?

**Comment by gsmet** [comment]
Hi.

It's in good way to be incorporated in the upcoming 3.14: https://github.com/quarkusio/quarkus/pull/42445 .
